paper.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "PyMedPhys: A community effort to develop an open standard library for Medical Physics in Python"
+tags:
+  - Python
+  - Medical Physics
+  - Radiation Therapy
+  - Diagnostic Imaging
+  - DICOM
+authors:
+  - name: Simon Biggs^[co-first author]
+    affiliation: 1
+  - name: Stuart Swerdloff^[co-first author]
+    affiliation: 2
+  - name: Matthew Jennings^[co-first author]
+    affiliation: 3
+  - name: Phillip Chlap^[co-first author]
+    affiliation: "4, 5"
+  - name: Jacob Rembish^[co-first author]
+    affiliation: 6
+affiliations:
+  - name: Radiotherapy AI, Australia
+    index: 1
+  - name: ELEKTA Pty Ltd, New Zealand
+    index: 2
+  - name: Royal Adelaide Hospital, Australia
+    index: 3
+  - name: University of News South Wales, Australia
+    index: 4
+  - name: Ingham Institute, Australia
+    index: 5
+  - name: NYU Langone Health, USA
+    index: 6
+date: 10 February 2022
+bibliography: paper.bib
+---
+
+# Summary
+
+PyMedPhys is an open-source medical physics Python library built by a community
+that values and prioritizes code sharing, review, improvement, and peer
+development. The overall goal of PyMedPhys is to provide medical physicists
+around the world with a free, open-source library of Python applications to
+simplify and enhance both their research and clinical work. It is inspired by
+the collaborative work of our physics peers in astronomy and the Astropy
+Project [@astropy].
+
+# Statement of need
+
+As the treatment delivery methods used in radiation oncology continue to
+improve, the complexity associated with ensuring precise and accurate delivery
+continues to increase. In response, many medical physicists are turning toward
+software solutions to help maximize efficiency and quality of care. It is
+common for medical physicists to develop their own in-house code for performing
+clinical tasks; many of which have been standardized through recommendations by
+organizations such as the American Association of Physicists in Medicine
+(AAPM). This can often lead to the "reinvention of the wheel", where physicists
+are wasting time writing code to serve a purpose that someone else has already
+accomplished.
+
+By creating and promoting an open-source repository, PyMedPhys aims to make the
+software solutions medical physicists have created easily accessible to others.
```

Comment:
perhaps *"PyMedPhys aims to improve the quality and accessibility of existing software solutions to problems faced by medical physicists"*?

I think it makes sense to focus on *medical physics-related applications* rather than *medical physicists*, noting that plenty of PyMedPhys' code was written by *non*-medical physicists, nor is it only useful to medical physicists.